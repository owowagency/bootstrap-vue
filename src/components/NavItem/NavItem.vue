<template>
    <li class="nav-item">
        <!-- @slot Displays the content of the nav item -->
        <slot />
    </li>
</template>

<docs>
```vue
<NavItem>
    <RouterLink :to="home">Home</RouterLink>
</NavItem>
```
</docs>
