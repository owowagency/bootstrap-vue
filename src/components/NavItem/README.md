# NavItem

```vue
<NavItem>
    <RouterLink :to="home">Home</RouterLink>
</NavItem>
```

## Slots

| Name    | Description                          | Bindings |
| ------- | ------------------------------------ | -------- |
| default | Displays the content of the nav item |          |
