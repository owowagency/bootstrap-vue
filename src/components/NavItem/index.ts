import NavItem from './NavItem.vue';

export default NavItem;
