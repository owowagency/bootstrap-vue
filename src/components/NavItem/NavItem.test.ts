import NavItem from '.';
import {describe} from 'vitest';

describe('template', () => {
    componentRenderTest(NavItem);

    componentSlotRenderTest(NavItem);
});
