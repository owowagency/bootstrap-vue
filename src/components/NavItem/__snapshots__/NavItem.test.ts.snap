// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`template > renders default 1`] = `
"<li class=\\"nav-item\\">
  <!-- @slot Displays the content of the nav item -->
</li>"
`;
